pkg/manifest/manifest_persitence.go

```diff
@@ -39,8 +39,9 @@ type authSecret struct {
 }
 
 type oAuth struct {
-	ClientID     authSecret `yaml:"clientId"`
-	ClientSecret authSecret `yaml:"clientSecret"`
+	ClientID      authSecret `yaml:"clientId"`
+	ClientSecret  authSecret `yaml:"clientSecret"`
+	TokenEndpoint url        `yaml:"token_endpoint,omitempty"`
```

Comment:
please make that `tokenEndpoint` , so we consistently use the same case in the YAML

Reply:
done